<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Principal</class>
 <widget class="QMainWindow" name="Tela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GeMapCom</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1276</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_tela_comparacao">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPARAR SEQUENCIAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_tela_converter">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>CONVERTER SEQUENCIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_tela_outras">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>OUTRAS FERRAMENTAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_sair">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
